src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -77,35 +78,40 @@
 				placement are rendered depending on the current page -->
 				<div class="navigation">
 					<!-- First page -->
-					<button v-if="page===0 && isPublic"
-						class="navigation__button navigation__button-left "
+					<Button v-if="page===0 && isPublic"
+						class="navigation__button navigation__button-left"
```

Comment:
Again, there should be no need to target buttons with classes once we've updated it to the component